### ISSUE TITLE
Shapley values for interactions between variables

### ISSUE THREAD

**Issue by langbe** [NorskRegnesentral/shapr#316]
Will shapley interaction values be supported in the next release?

**Comment by martinju** [comment]
Hi! We have not planned to include it. However, technically it should not be very difficult. I will therefore write it down on the lengthy "nice to have" list. Thanks!

**Comment by langbe** [comment]
Thank you. This is much appreciated, having the interactions could be very helpful for certain investigations. Another thought: since run-time will be a potential issue, it could make sense to have an option to specify only a certain subset of variables. Also, in some situations one might only be interested in a very specific order of interactions, e.g. between x1 and x2, x1 and x3, x1 and x4 etc. (ie interactions only to a certain reference variable).

**Comment by martinju** [comment]
I have little experience with Shapley interaction values, so I guess I have to read up on it, but is what you are suggesting actually possible? We are using the least-squares analog of the Shapley formula (KernelSHAP) for estimating the Shapley values in this package, and there I can't really decide to leave out any of the Shapley values from the computation...?

**Comment by langbe** [comment]
Right, I just had run-time in mind. I agree with you, I don't think it will work to easily restrict it to a subset of variables.
